plothist/hep_plotters.py

```diff
@@ -121,6 +121,29 @@ def compare_data_mc(
             uncertainties_high**2,
         )
         data_hist[:] = np.stack([data_hist.values(), data_variances], axis=-1)
+    elif comparison_kwargs["comparison"] == "ratio":
+        if comparison_kwargs["ratio_uncertainty"] == "split":
+            np.seterr(divide="ignore", invalid="ignore")
```

Comment:
what about `np.seterr(all="ignore")`

Reply:
No, we do not want to mute other potential warnings